README.md

```diff
@@ -57,13 +57,29 @@ This command will:
 
 Note: the `-f / --filename` flag can be optionally used to point to a specific data dump `.sql` file to use to restore. The way the snapshots are dated means that one will only end up downloading one copy of the data per-day, both with and without change log data.
 
-4. Start the Vision Zero Editor
+See [here](docs/local_dev.md) for in-depth documentation about the `vision-zero` helper.
+
+4. Configure the Vision Zero editor. Navigate to the `/editor` subdirectory and create a file called `.env.local`, and save the contents of the 1Password item called **Vision Zero Editor (VZE) Environment File** into this file.
+
+5. Activate your Node environment - requires [`nvm`](https://github.com/nvm-sh/nvm).
 
 ```shell
-vision-zero vze-up
+nvm use
 ```
 
-See [here](docs/local_dev.md) for in-depth documentation about the `vision-zero` helper.
+6. Install required package
```

Comment:
do we need to have instructions for starting the VZE both here in the base readme and in the editor readme? if the VZV instructions are just in the VZV folder than maybe we should adopt the same pattern for the editor

Reply:
I think it makes sense to have the steps to spin up the vze in this quickstart readme -- maybe the part on line 25 
>The helper script, `vision-zero`, makes it easy to spin up your local Vision Zero stack
should be updated to say that is only spins up the backend? I could go either way